### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by thephred** [crossplane/crossplane.github.io#60]
We have a security alert that suggests we "Upgrade kramdown to version 2.3.0 or later". Will do a pass on the deps and get things updated.

**Comment by jbw976** [comment]
Thank you for taking this on @thephred! If you learn any lessons about how to keep things up to date in the future going forward (e.g. the built in github dependabot) or have any suggestions for process improvements here, that would also be appreciated 😀

**Comment by thephred** [comment]
I didn't know about dependabot. Looking at it it seems like it might work but someone really ought to check out the PR it creates and check that `preprocess.js` and the site still works.

When I just did that manually I ran:
```
bundle update
```
To get new versions of RubyGems.

I ran manually updated the versions in `package.json` to be the latest for the two packages in there. and then ran:
```
npm install
```

Then I run:
```
node preprocess.js
```

To make sure that that still works properly (check the `_data/versions.json`.

Then I ran:
```
make run_docs_local
```

and verified that the site looked ok locally @ http://localhost:4000

I also had to not commit the changes to `_data/versions.json` because it adds a `local` version that is not needed.
